src/Components/Navigation.tsx

```diff
@@ -0,0 +1,25 @@
+import { useNavigate } from "react-router";
+import { Link } from "react-router-dom";
+import Logo from "../Assets/logo.png";
+
+export const Navigation = () => {
+    const navigate = useNavigate();
+
+  return (
+    <nav className="flex justify-between items-center h-12 sticky top-0 mb-1 p-4 bg-blue-900 rounded-b-xl">
+      <div className="flex justify-center items-center cursor-pointer " onClick={() => navigate("/")}>
```

Comment:
Not a suggestion, just a question, was using <Link> here giving a problem? I'm asking because you used navigate function()

Reply:
`<Link>` was giving me issues with styling. Hence I used this as an alternative.